### DISCUSSION TITLE
Automatically sizing images and linking to Table?

### DISCUSSION THREAD

**Discussion by oguruma1218** [supabase/supabase#14413] [Questions]
I'm looking to replace a headless Worpress instance with Supabase. Wordpress has a feature that will create mulitple copies of uploaded images in different sizes when an image is uploaded (so you can load an appropriately sized image for bandwidth/performance benefits).

If I have a Table "Posts" with:
post_id
post_featured_image_small_url
post_featured_image_medium_url
post_featured_image_full_url

If I have a front-end on the app that will create a new "Post" with a "featured_image," is it possible to have Supabase make the differently-sized images, add them to a Bucket, and then insert the various urls for the various sizes?

**Comment by GaryAustin1** [comment]
Supabase has image transformations as a url option when you use the image.
https://supabase.com/docs/guides/storage/image-transformations
You specify the image sizing in the URL when the image is fetched.
The image is uploaded in whatever the original size you decide to upload.

This is available on the Pro tier.

In your example you would store the path to the image 1 time.
Then when you use it, you would generate the url to the image with the transformation information as parameters.
